base/array.jl

```diff
@@ -956,6 +956,23 @@ function intersect(v1, vs...)
     ret
 end
 
+"""
+    isdisjoint(v1, v2) -> Bool
+
+Returns whether the collections `v1` and `v2` are disjoint.
+"""
+function _isdisjoint(v1, v2)
+    for v in v1
+        if v in v2 return false end
```

Comment:
These kinds of one-liners are usually written `v in v2 && return false` elsewhere in Base.